frame/timestamp/src/lib.rs

```diff
@@ -161,17 +157,8 @@ decl_module! {
 		/// `MinimumPeriod`.
 		///
 		/// The dispatch origin for this call must be `Inherent`.
-		///
-		/// # <weight>
-		/// - `O(T)` where `T` complexity of `on_timestamp_set`
-		/// - 1 storage read and 1 storage mutation (codec `O(1)`). (because of `DidUpdate::take` in `on_finalize`)
-		/// - 1 event handler `on_timestamp_set` `O(T)`.
-		/// - Benchmark: 7.678 (min squares analysis)
-		///   - NOTE: This benchmark was done for a runtime with insignificant `on_timestamp_set` handlers.
-		///     New benchmarking is needed when adding new handlers.
-		/// # </weight>
```

Comment:
Why removing all this ? I would still prefer keeping it, this is useful to check if benchmarks are missing some complexity, I would just remove
```
		/// - Benchmark: 7.678 (min squares analysis)
		///   - NOTE: This benchmark was done for a runtime with insignificant `on_timestamp_set` handlers.
		///     New benchmarking is needed when adding new handlers.
```